tests/foreman/cli/test_discoveryrule.py

```diff
@@ -240,12 +284,15 @@ def test_positive_create_disabled_rule(self, discoveryrule_factory):
         rule = discoveryrule_factory(options={'enabled': 'false'})
         assert rule.enabled == 'false'
 
-    @pytest.mark.tier3
+    @pytest.mark.tier1
+    @pytest.mark.parametrize('host_name', **parametrized(invalid_hostnames_list()))
     @pytest.mark.parametrize('name', **parametrized(invalid_values_list()))
-    def test_negative_create_with_invalid_name(self, name, discoveryrule_factory):
-        """Create Discovery Rule with invalid names
+    def test_negative_create_rule_with_invalid_discovery_params(
+        self, name, host_name, discoveryrule_factory
+    ):
+        """Create Discovery Rule with invalid parameters.
```

Comment:
I don't think it's good practice to test all negative params into single test, as it could break in first with block but there are 3 more scenarios afterward which could get skipped, also there'd be unneccessary parametrization combinations due it, so would be better to keep this negative tests as it

Reply:
updated